WISE-Community/WISE
# Fixed issue inserting note into report

This is for #1293

How to test:
1. Add a note. It can be text or image or both
2. Open report
3. Try to insert note from step 1

The note from step 1 should be inserted into the report.
